templates/openstack/install.html

```diff
@@ -326,7 +129,7 @@ <h2>Multi-node OpenStack deployment</h2>
   </div>
   <div class="row">
     <div class="col-8">
-      <p>These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> too. However, in this case a clustering feature is used.</p>
+      <p>These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; in a snap. MicroStack is a pure upstream OpenStack distribution, designed for small scale and edge deployments, that can be installed and maintained with a minimal effort</p>
```

Comment:
```suggestion
      <p>These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; OpenStack in a snap. MicroStack is a pure upstream OpenStack distribution, designed for small scale and edge deployments, that can be installed and maintained with a minimal effort</p>
```
Doesn't match copydoc